docs/core/entity/sensor.md

```diff
@@ -64,6 +64,7 @@ If specifying a device class, your sensor entity will need to also return the co
 | volatile_organic_compounds | µg/m³ | Concentration of volatile organic compounds
 | voltage | V | Voltage
 | volume | L, mL, gal, fl. oz., m³, ft³ | Generic volume
+| weight | kg, g, mg, µg, oz, lb | Generic mass
```

Comment:
Should this be "Generic weight" or include a note similar to https://github.com/home-assistant/core/blob/9bd21172400a444d23faf70cb492b8cc03841457/homeassistant/components/sensor/__init__.py#L195